src/js/Properties.js

```diff
@@ -243,7 +243,7 @@ axs.properties.findTextAlternatives = function(node, textAlternatives, opt_recur
 
     // 1. Skip hidden elements unless the author specifies to use them via an aria-labelledby or
     // aria-describedby being used in the current computation.
-    if (!recursive && !opt_force && axs.utils.isElementOrAncestorHidden(element))
+    if (!opt_force && axs.utils.isElementOrAncestorHidden(element))
```

Comment:
Sorry got sidetracked on another project :blush: 

The case I have added a test for is this:
```
<a href="#" aria-labelledby="foobar">
    <span id="foobar" aria-hidden="true">X</span>
    <span>Close this window</span>
</a>
```
To answer your question, it works by finding its text alternative not from line 475 but from here:
https://github.com/GoogleChrome/accessibility-developer-tools/blob/48a69f03f644f9f83b3370e4a72d07b6d704bb1e/src/js/Properties.js#L477

Actually I think line 477 might be a bug anyway and should read like this instead:
`labelledbyText.push(labelledby.text);` Which would then, indeed, result in this code not working. A test for this would be:
```
<span id="foobar" aria-label="Learn more about trout fishing"></span>
<a href="#" aria-labelledby="foobar"></a>
```

I think `opt_force` should always be passed in from the getTextFromAriaLabelledby method. A case to prove this would be:
```
<span id="foobar" aria-label="Learn more about trout fishing" aria-hidden="true"></span>
<a href="#" aria-labelledby="foobar"></a>
```
Interestingly:
ChromeVox does find the text alternative
Voiceover doesn't find the text alternative
chrome://accessibility/ doesn't find the text alternative